tests/functional/r/raise/raising_format_tuple.py

```diff
@@ -12,23 +12,23 @@ def bad_percent(arg):
 
 def good_percent(arg):
     '''Instead of passing multiple arguments, format the message'''
-    raise KeyError('Bad key: %r' % arg)
```

Comment:
This file should probably not be touched as the formatting of the string seems to be a test in itself ?

Reply:
True. Going to fix now!